app/javascript/controllers/application.js

```diff
@@ -1,9 +1,25 @@
-import { Application } from "@hotwired/stimulus"
+import { Application } from "@hotwired/stimulus";
 
-const application = Application.start()
+const application = Application.start();
 
 // Configure Stimulus development experience
-application.debug = false
-window.Stimulus   = application
+application.debug = false;
+window.Stimulus = application;
 
-export { application }
+export { application };
```

Comment:
Theres a better way to do this with stimulus which comes already with new rails version
[Checkout this blog](https://thecodest.co/blog/need-to-use-the-common-js-frameworks-in-your-rails-app/)